ee/codegen/src/generators/workflow.ts

```diff
@@ -545,7 +545,16 @@ export class Workflow {
       const allEdges = this.getEdges();
       allEdges.forEach((edge) => {
         if (!usedEdges.has(edge)) {
-          this.unusedEdges.add(edge);
+          if (
+            this.workflowContext.globalNodeContextsByNodeId.has(
```

Comment:
[nit] Instead of `this.workflowContext.globalNodeContextsByNodeId` (which is a derived field), let's use the original data by creating a set of node ids with `this.workflowContext.workflowRawData.nodes` below